target/test-classes/runners/mainFunctionsSuite.xml

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Main calculator functions" parallel="methods">
```

Comment:
Try to use another parallel mode, for instance, classes or something else. Not only methods.